src/main/app/controller/PostController.ts

```diff
@@ -52,11 +53,18 @@ export class PostController<T extends AnyObject> {
       req.session.errors = errors;
       nextUrl = req.url;
     } else {
+      nextUrl = getNextStepUrl(req, req.session.userCase);
+      if (nextUrl === JURISDICTION_INTERSTITIAL_URL) {
```

Comment:
rather than doing this here, find the field from that page and then do this logic in the to-api-format